content/blog/twitter-data-download-explanation.en.md

```diff
@@ -0,0 +1,906 @@
+{
+    "title": "Explaining Twitter's Data Download Feature.",
+    "type": "blog",
+    "slug": "twitter-data-download-explanation",
+    "date": "2019-10-17T16:58:19+02:00",
+    "description": "Twitter provides the option to download the user's personal data from their settings. This post explains the process to get the data as well as the meaning of the data provided.",
+    "featured_image": "/card-icons/Twitter_Black.svg",
+    "tags": [ "twitter", "gdpr", "data", "download", "export" ],
+    "authors": [ "xndr" ]
+}
+
+<img class="offset-image offset-image-left" src="/card-icons/Twitter_Black.svg" style="height: 200px;">
+
+Twitter is one of the top social media sites of our time. It has a high number of daily users and popular celebrities, sportstars and politicians use it as a method of communication with their fan base. It also serves as the source of daily news and entertainment for a lot of users. Using the insights from the world of hashtags, Twitter along with other brands and influencers have created an entirely new field of marketing. These insights are driven by the data collected by Twitter, which is quite extensive. 
+
+Twitter collects the users personal data to create and serve ads to them. For this purpose, it collects relevant user data and stores it to determine user interest and likes. This data can be a treasure trove of information on any frequent user. If the user opts in, the data is also shared with other partners as well. Twitter also stores a profile on any non logged-in user as well to serve ads based on their browsing habits. This is done on the device only and this data changes once the user logs in. The data stored is used to infer identity and serve ads as well as can be shared with other partners of Twitter.
+This data can be viewed by the user thanks to Article 15 GDPR which gives the right to data access. Know more about your rights in [this article]({{< ref "/blog/your-gdpr-rights" >}}). The method to access user data from such sites usually is to use the [generator]({{< ref "/generator" >}}). 
+
+
+If you go to [Twitter's settings page](https://twitter.com/settings/your_twitter_data) for user data without any user being logged in, you'll see a page that looks a bit like this.      
+  
+  
+  
+  
+
+
+{{< img src="/img/blog/twitter-data-explanation/non-login-twitter.png" caption="" >}}  
+
+
+<a id="twitter-logged-in-procedure"></a>
```

Comment:
You can do this via a heading anchor: https://gohugo.io/content-management/cross-references/#hugo-heading-anchors